From 51b20e3114a91d612b163b35cbf8e5ae9a78d714 Mon Sep 17 00:00:00 2001
From: BrLi <brli@chakralinux.org>
Date: Fri, 27 May 2022 22:32:33 +0800
Subject: [PATCH] prepare.sh: remove yarn lines

---
 prepare_vscode.sh | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/prepare_vscode.sh b/prepare_vscode.sh
index 0334d00..4317d9c 100755
--- a/prepare_vscode.sh
+++ b/prepare_vscode.sh
@@ -34,23 +34,6 @@ done
 
 set -x
 
-if [[ "${OS_NAME}" == "osx" ]]; then
-  CHILD_CONCURRENCY=1 yarn --frozen-lockfile --ignore-optional
-  npm_config_argv='{"original":["--ignore-optional"]}' yarn postinstall
-elif [[ "${npm_config_arch}" == "armv7l" || "${npm_config_arch}" == "ia32" ]]; then
-  # node-gyp@9.0.0 shipped with node@16.15.0 starts using config.gypi
-  # from the custom headers path if dist-url option was set instead of
-  # using the config value from the process. Electron builds with pointer compression
-  # enabled for x64 and arm64, but incorrectly ships a single copy of config.gypi
-  # with v8_enable_pointer_compression option always set for all target architectures.
-  # We use the force_process_config option to use the config.gypi from the
-  # nodejs process executing npm for 32-bit architectures.
-  export npm_config_force_process_config="true"
-  CHILD_CONCURRENCY=1 yarn --frozen-lockfile
-else
-  CHILD_CONCURRENCY=1 yarn --frozen-lockfile
-fi
-
 mv product.json product.json.bak
 
 # set fields in product.json
-- 
2.36.1
